### PR TITLE
Daily invalid entries

### PR DESCRIPTION
**Pull request by jniechcial**
It introduces rake tasks for sending notifications about:
* long lunch entry (over 30 minutes)
* entry without project

### CODE REVIEW

**Comment by marcinjak** [comment]
LGTM :+1:

File: templates/empty_project.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hej,
```
Comment (dmp0x7c5) [inline templates/empty_project.txt]: Shouldn't it be in English? We've got a few (2?) non-polish speakers in the company :)

Comment (jniechcial) [inline templates/empty_project.txt]: Let's see how it works and change the content if it gives results ;) But true, it should be in English

Comment (dmp0x7c5) [inline templates/empty_project.txt]: 👍

**Pull request merged**